dev-tools/templates/.golangci.yml

```diff
@@ -92,21 +90,12 @@ linters-settings:
   goimports:
     local-prefixes: github.com/elastic
 
-  gomodguard:
```

Comment:
Yeah this is tricky, I would like to migrate to the standard library functionality here eventually.

I really like your suggestion of a compatibility layer, it would be much easier to just change the import path with something that redirects to the standard `errors` package than fixing every single call to `Wrap`.

We have relatively few usages of `errors.Cause` that should be easy to manually remove.

I'm going to put this block back in, and create an issue to 1) remove all uses of `errors.Cause` and 2) introduce a compatibility layer that handles the `nil` error case.

Reply:
I created https://github.com/elastic/beats/issues/31702 to track trying to remove this package and fix the lint errors plaguing everyone.